src/integration-test/java/com/gradle/exportapi/DatabaseBaseIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package com.gradle.exportapi;
+
+import org.junit.After;
+import org.junit.Before;
+import org.knowm.yank.PropertiesUtils;
+import org.knowm.yank.Yank;
+
+import java.util.Properties;
+
+public abstract class DatabaseBaseIntegrationTest {
+
+    @Before
+    public void setup() {
+        Yank.setThrowWrappedExceptions(true);
+        Properties dbProps = PropertiesUtils.getPropertiesFromClasspath("POSTGRES.properties");
```

Comment:
I will change the default to a test_db.properties to reduce the chance of it happening.  Then yes, once I add a CI job it will spin up DBs in docker.